hbase-server/src/test/java/org/apache/hadoop/hbase/io/hfile/TestPrefetchRSClose.java

```diff
@@ -114,6 +117,15 @@ public void testPrefetchPersistence() throws Exception {
     // should exist.
     assertTrue(new File(testDir + "/bucket.persistence").exists());
 
+    HRegionServer regionServingRS = cluster.getRegionServer(1).getRegions(tableName).size() == 1
+      ? cluster.getRegionServer(1)
+      : cluster.getRegionServer(0);
+
+    Admin admin = TEST_UTIL.getAdmin();
+    List<String> prefetchedFilesList =
+      admin.getPrefetchedFilesList(regionServingRS.getServerName());
+    assertEquals(1, ((List<?>) prefetchedFilesList).size());
```

Comment:
Nit: (List<?>) Redundant cast can be removed as prefetchedFilesList type is List<String>